UPDATING.md

```diff
@@ -6,12 +6,33 @@ The 2.0 version of ably-python introduces our first Python realtime client. For
 
 In addition to this, we have also made some minor breaking changes, these include:
 
+  - Added mandatory version param to `AblyRest.request`
+  - Changed return type of `AblyRest.stats`
   - Removed `Auth.authorise` (in favour of `Auth.authorize`)
   - Removed `Options.fallback_hosts_use_default`
   - Removed `Crypto.get_default_params(key)` signature.
   - Removed the `client_id` and `extras` kwargs from `Channel.publish`
   - Calling `channels.release()` no longer raises a `KeyError` if the channel does not yet exist
 
+### Added mandatory version param to `AblyRest.request`
+
+If you were using the generic `request` method to query the Ably REST API, you will now need to pass a version string as the third parameter. The version string represents the version of the Ably REST API to use, allowing you to upgrade to newer versions of REST endpoints as soon as they are released.
+
+```python
+await rest.request("GET", "/time", "3.0")
```

Comment:
It would probably make more sense for this to be a version that's currently listed in the dropdown on the [REST API reference](https://ably.com/docs/api/rest-api) — I think it will confuse users otherwise.

```suggestion
await rest.request("GET", "/time", "1.2")
```

Reply:
good point, updated